marklogic-data-hub-central/ui/src/components/modeling/property-table/property-table.test.tsx

```diff
@@ -240,6 +240,10 @@ describe('Entity Modeling Property Table Component', () => {
     fireEvent.change(wildcardCheckbox, { target: { checked: true } });
     expect(wildcardCheckbox).toBeChecked();
 
+    const facetableCheckbox = screen.getByLabelText('Facetable');
```

Comment:
I think the code should definitely use the term "facetable" to avoid confusion over what "facet" means based on the context it's used in. But verify with @yasht2010 that it's okay for the UI to say "Facetable" - that's a change in the design. I don't have any issue with the UI saying e.g. "Facet" and "Sort" - in that context, those can be seen as verbs. But in the entity model itself, I think we want adjectives.

Reply:
I'd prefer to use the word "Facet" in the UI. "Facetable" is not a valid word.

@eltesoro-ml

Reply:
How about...
    `const isFacetCheckbox = screen.getByLabelText('Use as a facet');`

Reply:
"facetable" has use in other products - for example, Azure uses it in their docs - https://docs.microsoft.com/en-us/azure/search/search-filters-facets , along with "filterable".

FWIW, I believe "facetable" is commonly used in the field too - I think it's common for a consultant to say e.g. "Let's make that property facetable" (often as a substitute for "searchable"). 

It's not in the dictionary, but - well, that hasn't stopped Microsoft from using it. :)

Reply:
https://64.media.tumblr.com/7bc8be17beeed1dd44aa4e18b5e3e825/tumblr_inline_okb99j5SEa1sstivl_500.png
"FACETABLE ... when face palm just isn't enough."